pkg/broker/cel.go

```diff
@@ -57,87 +45,64 @@ func (r *Receiver) filterEventByCEL(ts *eventingv1alpha1.TriggerSpec, event *clo
 		return false, err
 	}
 
```

Comment:
+1 to caching the `cel.Program` by expression string since they parse / check is considerably more expensive than the evaluation, like orders of magnitude more.

Reply:
Done.